test failure

the CI has been failing for a while, anyone knows whats going on?
https://github.com/LinuxCNC/linuxcnc/actions/runs/6683612209/job/18159988194#step:7:9653

````
dh_installdocs --doc-main-package=linuxcnc-uspace --package=linuxcnc-doc-de
dh_installdocs: error: Cannot find (any matches for) "usr/share/doc/linuxcnc/LinuxCNC_Developer_de.pdf" (tried in ., debian/tmp)

	install -d debian/linuxcnc-doc-de/usr/share/doc/linuxcnc-doc-de
make[1]: *** [debian/rules:109: override_dh_installdocs-indep] Error 25
````
I am wondering why the .de docs get special treatment there? Especially as they appear to be the problem.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -33,7 +33,7 @@ jobs:
         sudo apt-get install -y eatmydata
         eatmydata ./scripts/travis-install-build-deps.sh
         eatmydata curl -O https://snapshot.debian.org/archive/debian/20220716T154603Z/pool/main/p/po4a/po4a_0.67-2_all.deb
-        sudo eatmydata apt install ./po4a_0.67-2_all.deb
+        sudo eatmydata apt install --yes ./po4a_0.67-2_all.deb
         cd src
         eatmydata ./autogen.sh
         eatmydata ./configure --with-realtime=uspace --disable-check-runtime-deps
@@ -60,7 +60,7 @@ jobs:
         eatmydata ./scripts/travis-install-build-deps.sh
         sudo eatmydata apt-get install -y clang
         eatmydata curl -O https://snapshot.debian.org/archive/debian/20220716T154603Z/pool/main/p/po4a/po4a_0.67-2_all.deb
-        sudo eatmydata apt install ./po4a_0.67-2_all.deb
+        sudo eatmydata apt install --yes ./po4a_0.67-2_all.deb
         cd src
         eatmydata ./autogen.sh
         CC=clang CXX=clang++ eatmydata ./configure --with-realtime=uspace --disable-check-runtime-deps
@@ -85,7 +85,7 @@ jobs:
         ./scripts/travis-install-build-deps.sh
         sudo apt-get install -y eatmydata
         curl -O https://snapshot.debian.org/archive/debian/20220716T154603Z/pool/main/p/po4a/po4a_0.67-2_all.deb
-        sudo apt install ./po4a_0.67-2_all.deb
+        sudo apt install --yes ./po4a_0.67-2_all.deb
         cd src
         eatmydata ./autogen.sh
         eatmydata ./configure --with-realtime=uspace --disable-check-runtime-deps --enable-build-documentation=html
@@ -158,6 +158,8 @@ jobs:
         eatmydata debian/update-dch-from-git
         eatmydata scripts/get-version-from-git | sed -re 's/^v(.*)$/\1/' >| VERSION; cat VERSION
         eatmydata git diff
+        eatmydata curl -O https://snapshot.debian.org/archive/debian/20220716T154603Z/pool/main/p/po4a/po4a_0.67-2_all.deb
+        eatmydata apt install --yes ./po4a_0.67-2_all.deb
         eatmydata apt-get --yes --quiet build-dep --arch-only .
         eatmydata debuild -us -uc --build=any
     - name: Test debian packages
@@ -239,6 +241,8 @@ jobs:
         eatmydata debian/update-dch-from-git
         eatmydata scripts/get-version-from-git | sed -re 's/^v(.*)$/\1/' >| VERSION; cat VERSION
         eatmydata git diff
+        eatmydata curl -O https://snapshot.debian.org/archive/debian/20220716T154603Z/pool/main/p/po4a/po4a_0.67-2_all.deb
+        eatmydata apt install --yes ./po4a_0.67-2_all.deb
         eatmydata apt-get --yes --quiet build-dep --indep-only .
         eatmydata debuild -us -uc --build=all
     - name: Test install debian packages

